js-weather/index.js

```diff
@@ -0,0 +1,62 @@
+// 天気のデータを取得
+function getWeatherData(cityData) {  
+  let targetCityName = cityData;
+  let appId = "b40c5bbc82d4fa2558581d2a6898068d"
+  
+  const requestUrl = "https://api.openweathermap.org/data/2.5/weather?APPID=" + appId + "&lang=ja&units=metric&q=" + targetCityName + ",jp;"
+  
+  // Ajax通信用のオブジェクトを作成
+  let xhr = new XMLHttpRequest();
+  
+  // 通信方式とURLを設定
+  xhr.open("GET",requestUrl);
+  
+  // 通信を実行する
+  xhr.send();
+  
+  // 通信ステータスが変わったら実行される関数
+  xhr.onreadystatechange = function() {
+    // 通信が完了
+    if(xhr.readyState == 4){
+      ShowTodaysWeather(xhr.responseText);
+    }
+  }
```

Comment:
```suggestion
  };
```
こことか文末のセミコロンが抜けていますね。
JSはセミコロン無くても大抵動きますが、セミコロンありならありで統一しましょう。